docusaurus/docs/learn/quickstarts/network/help/upgrade-quickstart-network.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Upgrade a Quickstart Network
+id: upgrade-quickstart-network
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This document explains the process for backing up, and upgrading your Ziti Controller and Ziti Router(s).
+
+<Tabs groupId="backup-network">
+<TabItem value="non-docker-quickstart" label="Non-Docker Quickstart">
+
+## Backup Existing Network
```

Comment:
i feel like both tabs should have the "Backup Existing Network" title but docker shows: "Docker Users"... I know I'm a docker user, I just clicked that tab... :)